packages/files/file.pony

```diff
@@ -212,19 +203,19 @@ class File
       return FileError
     end
 
-    
+
   fun valid(): Bool =>
     """
     Returns true if the file is currently open.
     """
-    not _handle.is_null()
+    not _fd == -1
 
   fun get_fd(): I32 ? =>
     """
     Returns the underlying file descriptor.
     Raises an error if the file is not currently open.
     """
-    if _handle.is_null() then
+    if _fd == -1 then
```

Comment:
I know `get_fd` existed before this PR, but it looks EXTREMELY dangerous to me. It also doesn't seem to be called from anywhere in the standard libraries. Can we remove this method?